index.html

```diff
@@ -313,7 +379,8 @@ <h6>CSS</h6>
         <!--End of Frameworks-->
 
       </div>
-    </section>
+    </div>
+
 
     <section class="contact-me background">
```

Comment:
- [ ] As the previous reviewer requested, kindly ensure that the text that is above the form is centered and has a specified width. This can be achieved by giving the text a width and centering it.

**This is how I am seeing this section:**

![image](https://user-images.githubusercontent.com/86318284/194526522-1bc6aa77-1744-4117-bb36-ca510876611e.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/194526616-e5e0880b-a7c0-4ec6-bae9-e27bbb475ab9.png)

- [ ] Also, kindly reduce the size of the background image